pkg/kube/inject/testdata/webhook/TestWebhookInject.patch

```diff
@@ -1,100 +1,29 @@
 [
   {
     "op": "add",
-    "path": "/spec/initContainers/-",
-    "value": {
-      "name": "istio-init",
-      "image": "example.com/init:latest",
-      "resources": {}
-    }
-  },
-  {
-    "op": "add",
-    "path": "/spec/containers/-",
-    "value": {
-      "name": "istio-proxy",
-      "image": "example.com/proxy:latest",
-      "resources": {}
-    }
-  },
-  {
-    "op": "add",
-    "path": "/spec/volumes/-",
+    "path": "/metadata/labels",
     "value": {
-      "name": "istio-envoy",
-      "emptyDir": {
-        "medium": "Memory"
-      }
+      "istio.io/rev": "",
+      "security.istio.io/tlsMode": "istio",
+      "service.istio.io/canonical-name": "",
+      "service.istio.io/canonical-revision": "latest"
     }
   },
   {
     "op": "add",
-    "path": "/spec/volumes/-",
+    "path": "/metadata/annotations",
     "value": {
-      "name": "istio-certs",
-      "secret": {
-        "secretName": "istio.default"
-      }
+      "prometheus.io/path": "/stats/prometheus",
+      "prometheus.io/port": "15020",
+      "prometheus.io/scrape": "true",
+      "sidecar.istio.io/status": "{\"version\":\"unit-test-fake-version\",\"initContainers\":null,\"containers\":null,\"volumes\":null,\"imagePullSecrets\":null}"
     }
   },
-  {
-    "op": "add",
-    "path": "/spec/imagePullSecrets",
```

Comment:
this `imagePullSecret` seems to be missing, too?

Reply:
Hmm.. I don't understand why this was here at all originally. The template doesn't have image pull secrets defined:
```
spec:
  initContainers:
    - name: c0
  containers:
    - name: c1
  volumes:
    - name: v0
```